sources/packages/web/src/views/aest/institution/DesignationAESTView.vue

```diff
@@ -188,7 +188,9 @@ export default {
             return designationLocation;
           });
       }
-      const response = await approveDenyDesignationModal.value.showModal();
+      const response = await approveDenyDesignationModal.value.showModal(
+        updateDesignationModel.value,
```

Comment:
is this file part of the pr? looks like  this page needs a refactors like `<header-navigator` should be inside `<full-page-container`